src/RenderWebGL.js

```diff
@@ -656,6 +655,79 @@ class RenderWebGL extends EventEmitter {
     }
 
     /**
+     * @typedef DrawableExtraction
+     * @property {Uint8Array} data Raw pixel data for the drawable
+     * @property {int} width Drawable bounding box width
+     * @property {int} height Drawable bounding box height
+     * @property {int} x The x coordinate relative to drawable bounding box
+     * @property {int} y The y coordinate relative to drawable bounding box
+     */
+
+    /**
+     * Return drawable pixel data and picking coordinates relative to the drawable bounds
+     * @param {int} drawableID The ID of the drawable to get pixel data for
+     * @param {int} x The client x coordinate of the picking location.
+     * @param {int} y The client y coordinate of the picking location.
+     * @return {?DrawableExtraction} Data about the picked drawable
+     */
+    extractDrawable (drawableID, x, y) {
+        const drawable = this._allDrawables[drawableID];
+        if (!drawable) return;
```

Comment:
This is a weakness in my understanding of JSDoc and JS convention:
Is it more conventional to just `return` here, or to explicitly `return null`?
Is `undefined` included in `{?DrawableExtraction}`?